dev/com.ibm.ws.jca_fat_dynamicConfig/test-applications/fvtweb/src/web/DynamicConfigTestServlet.java

```diff
@@ -0,0 +1,389 @@
+        // Note: we abuse the setLoginTimeout interface to trigger sending a message to the MDB
+        CommonDataSource ds = (CommonDataSource) new InitialContext().lookup("eis/cf");
+        DynaCfgMessageDrivenBean.messages.clear();
+        ds.setLoginTimeout(-1000);
+        ds.setLoginTimeout(0);
+        ds.setLoginTimeout(10);
+        ds.setLoginTimeout(1000);
+        if (!DynaCfgMessageDrivenBean.messages.isEmpty())
+            throw new Exception("Unepxected messages sent to MDB: " + DynaCfgMessageDrivenBean.messages);
+    }
+
+    /**
+     * Verify that an admin object of type java.util.Date is set to December 1, 2013.
+     *
+     * @param request  HTTP request
+     * @param response HTTP response
+     * @throws Exception if an error occurs.
+     */
+    public void testAdminObject_Date_2013_Dec_1(HttpServletRequest request, HttpServletResponse response) throws Exception {
+        Calendar cal = new GregorianCalendar();
+        cal.setTime((Date) new InitialContext().lookup("eis/myAdminObject"));
+        int year = cal.get(Calendar.YEAR);
+        if (year != 2013)
+            throw new Exception("Unexpected year: " + year + " for date " + cal);
+        int month = cal.get(Calendar.MONTH);
+        if (month != Calendar.DECEMBER)
+            throw new Exception("Unexpected month: " + month + " for date " + cal);
+        int dayOfMonth = cal.get(Calendar.DAY_OF_MONTH);
+        if (dayOfMonth != 1)
+            throw new Exception("Unexpected day of month: " + dayOfMonth + " for date " + cal);
+    }
+
+    /**
+     * Verify that an admin object of type java.util.Date is set to January 1, 2013.
+     *
+     * @param request  HTTP request
+     * @param response HTTP response
+     * @throws Exception if an error occurs.
+     */
+    public void testAdminObject_Date_2013_Jan_1(HttpServletRequest request, HttpServletResponse response) throws Exception {
+        Calendar cal = new GregorianCalendar();
+        cal.setTime((Date) new InitialContext().lookup("eis/myAdminObject"));
+        int year = cal.get(Calendar.YEAR);
+        if (year != 2013)
+            throw new Exception("Unexpected year: " + year + " for date " + cal);
+        int month = cal.get(Calendar.MONTH);
+        if (month != Calendar.JANUARY)
+            throw new Exception("Unexpected month: " + month + " for date " + cal);
+        int dayOfMonth = cal.get(Calendar.DAY_OF_MONTH);
+        if (dayOfMonth != 1)
+            throw new Exception("Unexpected day of month: " + dayOfMonth + " for date " + cal);
+    }
+
+    /**
+     * Verify that an admin object is available as type java.util.List implemented by java.util.LinkedList.
+     *
+     * @param request  HTTP request
+     * @param response HTTP response
+     * @throws Exception if an error occurs.
+     */
+    public void testAdminObject_List(HttpServletRequest request, HttpServletResponse response) throws Exception {
```

Comment:
Yes and no. While this test is not writing anything to the `response` all of these tests are being called via the overridden `doGet` method here: 

https://github.com/KyleAure/open-liberty/blob/6187544e4a6d01e9e8f40f82ae78542a59528055/dev/com.ibm.ws.jca_fat_dynamicConfig/test-applications/fvtweb/src/web/DynamicConfigTestServlet.java#L44-L63

The `doGet` method does require the use of request/response.  I would rather pass on that resquest/response onto the test methods so that in the future we can use them if we need them.